lib/assert.js

```diff
@@ -172,8 +172,8 @@ function getCode(fd, line, column) {
   let lines = 0;
   // Prevent blocking the event loop by limiting the maximum amount of
   // data that may be read.
-  let maxReads = 64; // bytesPerRead * maxReads = 512 kb
-  const bytesPerRead = 8192;
+  let maxReads = 2 ** 5; // bytesPerRead * maxReads = 512 kb
+  const bytesPerRead = 2 ** 14;
```

Comment:
Maybe it's just me, but I can more easily compare small powers of two if they're written in decimal